app/invites/client/admin/adminInvites.js

```diff
@@ -0,0 +1,90 @@
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+
+import { t, APIClient } from '../../../utils';
+
+Template.adminInvites.helpers({
+	isReady() {
+		if (Template.instance().ready != null) {
+			return Template.instance().ready.get();
+		}
+		return undefined;
+	},
+	invites() {
+		return Template.instance().invites.get();
+	},
+	isLoading() {
+		if (Template.instance().ready != null) {
+			if (!Template.instance().ready.get()) {
+				return 'btn-loading';
+			}
+		}
+	},
+	daysToExpire() {
+		const { expires, days, createdAt } = this;
+
+		if (days > 0) {
+			if (expires < new Date()) {
+				return t('Expired');
+			}
+
+			return Math.ceil((expires - createdAt) / (1000 * 60 * 60 * 24));
+		}
+
+		return t('Never');
+	},
+	maxUsesLeft() {
+		const { maxUses, uses } = this;
+
+		if (maxUses > 0) {
+			if (uses >= maxUses) {
+				return t('None');
+			}
+
+			return maxUses - uses;
+		}
+
+		return t('Unlimited');
+	},
+	creationDate() {
+		const { createdAt } = this;
+
+		return createdAt.toLocaleDateString();
+	},
+});
+
+Template.adminInvites.onCreated(function() {
+	const instance = this;
+	this.invites = new ReactiveVar([]);
+	this.ready = new ReactiveVar(false);
+
+	this.autorun(function() {
```

Comment:
Why is this inside the autorun? I can't see any reactivity that would lead to a refresh at any time.